integration/src/integration/java/pl/allegro/tech/hermes/frontend/KafkaNamesMapperConfiguration.java

```diff
@@ -6,7 +6,6 @@
 import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Profile;
 import pl.allegro.tech.hermes.common.kafka.KafkaNamesMapper;
-//todo change to frontend module
```

Comment:
**suggestion:** Can we create an issue for changing that module?

Reply:
reverted changes in this line